apps/web/src/components/TimestampToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Dispatch, FC, SetStateAction } from "react";
+
+import { Tooltip, TooltipContent, TooltipTrigger } from "~/components/Tooltip";
+
+export type TimestampFormat = "relative" | "absolute";
+
+type TimestampProps = {
+  format: TimestampFormat;
+  setFormat: Dispatch<SetStateAction<TimestampFormat>>;
```

Comment:
Please avoid using this type of prop as explained [earlier](https://github.com/Blobscan/blobscan/pull/602#discussion_r1818128469).

Let's update this prop to `onChange` of type `(timestampFormat: TimestampFormat) => void`

Reply:
Done [e9f4ba2](https://github.com/Blobscan/blobscan/pull/664/commits/e9f4ba2f39c641f2302074b3071acfc833e45a9c)